<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reservation_MainWindow</class>
 <widget class="QMainWindow" name="Reservation_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reservation_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>900</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLabel" name="name_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="name_plainTextEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>390</width>
        <height>45</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="reservationInfo_groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>500</width>
        <height>650</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QDateEdit" name="checkin_dateEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>200</width>
         <height>50</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="checkindate_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Check-in Date:</string>
       </property>
      </widget>
      <widget class="QLabel" name="checkoutdate_label">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Check-out Date:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="checkout_dateEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>100</y>
         <width>200</width>
         <height>50</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="numAdults_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>175</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Adults:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="numAdults_spinBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>75</width>
         <height>35</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
      <widget class="QLabel" name="numChildren_label">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>175</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Children:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="numChildren_spinBox">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>220</y>
         <width>75</width>
         <height>35</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="nightsParking_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>400</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Nights parking ($12.75/night)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="nightsParking_SpinBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>335</y>
         <width>75</width>
         <height>35</height>
        </rect>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
      <widget class="QGroupBox" name="roomSize_groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>400</y>
         <width>200</width>
         <height>150</height>
        </rect>
       </property>
       <property name="title">
        <string>Room Size:</string>
       </property>
       <widget class="QRadioButton" name="twoQueen_radioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>175</width>
          <height>35</height>
         </rect>
        </property>
        <property name="text">
         <string>2 Queen</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="oneKing_RadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>175</width>
          <height>35</height>
         </rect>
        </property>
        <property name="text">
         <string>1 King</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="roomView_groupBox">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>400</y>
         <width>200</width>
         <height>150</height>
        </rect>
       </property>
       <property name="title">
        <string>View:</string>
       </property>
       <widget class="QRadioButton" name="standardView_RadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>175</width>
          <height>35</height>
         </rect>
        </property>
        <property name="text">
         <string>Standard</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="atriumView_RadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>175</width>
          <height>35</height>
         </rect>
        </property>
        <property name="text">
         <string>Atrium</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="numDays_Label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>0 Night(s)</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="subtotal_label">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>580</y>
        <width>411</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Subtotal: $0.00</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPg1_pushButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>700</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="logo_label">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>30</y>
        <width>421</width>
        <height>551</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="roomCost_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>375</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>Room Cost: $0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="tax_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>375</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>Tax: $0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="parkingFee_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>375</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>Parking Fee: $0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="resortFee_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>375</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>Resort Fee: $0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalDue_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>450</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Total Due: $0.00</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="cardType_groupBox">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>351</width>
        <height>300</height>
       </rect>
      </property>
      <property name="title">
       <string>Card Type:</string>
      </property>
      <widget class="QRadioButton" name="VISA_radioButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>VISA</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="MasterCard_RadioButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>MasterCard</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Discover_RadioButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Discover</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="AmericanExpress_RadioButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>American Express</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="creditCard_lineEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>500</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="enterCardNum_Label">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>455</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter card number:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="prevPg2_PushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>700</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prev</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPg2_pushButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>700</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pay</string>
      </property>
     </widget>
     <widget class="QLabel" name="expirationDate_Label">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>575</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Expiration Date:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="cardExpireDate_DateEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>620</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>M/yyyy</string>
      </property>
     </widget>
     <widget class="QLabel" name="VISA_label">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>340</y>
        <width>87</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="masterCard_label">
      <property name="geometry">
       <rect>
        <x>647</x>
        <y>340</y>
        <width>87</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="discover_label">
      <property name="geometry">
       <rect>
        <x>734</x>
        <y>340</y>
        <width>87</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="americanExpress_label">
      <property name="geometry">
       <rect>
        <x>821</x>
        <y>340</y>
        <width>87</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="reservationDetails_groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>125</y>
        <width>941</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>Reservation Details</string>
      </property>
      <widget class="QLabel" name="checkInDatePg3_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>681</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Check-in Date:</string>
       </property>
      </widget>
      <widget class="QLabel" name="checkOutDatePg3_Label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>681</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Check-out Date:</string>
       </property>
      </widget>
      <widget class="QLabel" name="roomTypePg3_Label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>691</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Room Type:</string>
       </property>
      </widget>
      <widget class="QLabel" name="parkingPg3_Label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>691</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Parking:</string>
       </property>
      </widget>
      <widget class="QLabel" name="guestsPg3_Label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>330</y>
         <width>691</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Guests:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="paymentDetailsPg3_groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>585</y>
        <width>700</width>
        <height>250</height>
       </rect>
      </property>
      <property name="title">
       <string>Payment Details:</string>
      </property>
      <widget class="QLabel" name="totalCostPg3_Label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>681</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Total Cost:</string>
       </property>
      </widget>
      <widget class="QLabel" name="cardNumPg3_Label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>681</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Paid with card ending in:</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="exit_pushButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>750</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>700</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Thank you for choosing Island Resorts!</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
